external/llvm-project/mlir/lib/Dialect/GPU/Transforms/SerializeToHsaco.cpp

```diff
@@ -288,7 +288,7 @@ SerializeToHsacoPass::translateToLLVMIR(llvm::LLVMContext &llvmContext) {
 
     // This constant must always match the default code object ABI version
     // of the AMDGPU backend.
-    addControlConstant("__oclc_ABI_version", 500, 32);
```

Comment:
Mind to leave a comment on where shall one find this number from AMDGPU backend?